### PR TITLE
Add checking if user has company

### PR DESCRIPTION
**Pull request by konradko**

### CODE REVIEW

File: enrolment/tests/test_views.py [hunk 1]
```diff
@@ -306,89 +326,116 @@ def test_company_profile_details_logs_missing_sso_user(client, rf):
...
+)
+def test_enrolment_views_use_correct_template(client, rf):
     request = rf.get(reverse('register'))
     request.sso_user = mock.Mock(id=1, email="test@example.com")
     request.session = client.session
     view_classes = [
-        CompanyDescriptionEditView,
-        EnrolmentView,
-        CompanyProfileEditView
+        EnrolmentView
```
Comment (richtier) [inline enrolment/tests/test_views.py]: why this change?

File: enrolment/tests/test_views.py [hunk 2]
```diff
@@ -227,77 +234,90 @@ def test_company_profile_edit_api_client_call(
     request = rf.get(reverse('company-detail'))
     request.sso_user = mock.Mock(id=1, email="test@example.com")
```
Comment (richtier) [inline enrolment/tests/test_views.py]: instead of creating a request here you can use the `sso_request` fixture.

Comment (konradko) [inline enrolment/tests/test_views.py]: cool

File: enrolment/tests/test_views.py (hunk 1 above)
Comment (konradko) [inline enrolment/tests/test_views.py]: Enrolment view does not require use to have company, Company edit views do
